README.md

```diff
@@ -166,7 +166,34 @@ return this.infinityModel("product", { perPage: 12, startingPage: 1,
                                        category: "furniture" });
 ```
 
-Bound Parameters has been DEPRECATED as the third argument to `infinityModel`.  
+**Extending infinityModel**
+
+You can customize your infinityModel through configuring it like so and pass as the third argument to infinityModel:
+
+```js
+const ExtendedInfinityModel = InfinityModel.extend({
+  global: service(),
+  buildParams() {
+    let params = this._super(...arguments);
+    params['category_id'] = get(this, 'global.categoryId');
+    return params;
+  }
+});
+
+export default Route.extend({
+  global: service(),
+  categoryId: computed('global.categoryId', function() {
+    return get(this, 'global.categoryId');
+  }),
+  model() {
+    this.infinityModel('product', {}, ExtendedInfinityModel);
+  }
+});
+```
+
+**Bound Parameters** 
+
+DEPRECATED as the third argument to `infinityModel`.  
 Please include as second argument to infinityModel.
```

Comment:
As of 1.0+, passing a third parameter to represent Bound Parameters is deprecated.  All valid use cases of this feature should now be ported to the [Extended Infinity Model pattern][Extending infinityModel].

Bound Params were introduced as a way of dynamically fetching data over time - the query params passed to the server would be dictated by a property (computed or otherwise) on the route level, that was evaluated at the request time.

This design has always felt a little off - using computed properties on the Route level is an uncommon (and thus non-ergonomic) pattern in Ember.  As users have requested more features in Ember Infinity, we've realized it's more important to provide a flexible primitive that can be manipulated and extended in a Ember-esque way.  This opens Ember Infinity up to a great deal more use cases, while also providing a path forward to those using the pre 1.0 version of Bound Params.